new_dawn_server/actions/api/resources.py

```diff
@@ -65,8 +65,8 @@ def obj_create(self, bundle, **kwargs):
         super(UserActionResource, self).obj_create(bundle).obj.save()
         if bundle.data.get("action_type") == ActionType.LIKE.value:
             NotificationService().send_notification([str(bundle.data.get("user_to_id"))], message="You are liked")
-            if UserAction.objects.filter(user_to_id=bundle.data.get("user_from_id"),
-                                         user_from_id=bundle.data.get("user_to_id"),
+            if UserAction.objects.filter(user_to__username=bundle.data.get("user_from_id"),
+                                         user_from__username=bundle.data.get("user_to_id"),
```

Comment:
This is reading user_to_id from client, but how is it used in username?

Reply:
In the future, client's notion of "user_id" should all become "username", and we will no longer use the real id anymore

Reply:
And username will be phone number

Reply:
I see